src/Configuration/TableDetailParameters.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorCommon\Configuration;
+
+class TableDetailParameters
```

Comment:
To je takový divný název, ne? Není to spíš FullyQualifiedTableName nebo tak něco?

Reply:
Plus mám pocit, že ne vždycky je potřeba schema specifikovat, ne? Myslím, že v MSSQL je defaultně "dbo" nebo tak něco. Nebude to vadit downstream @pivnicek ?

Reply:
nah, it's ok imho.  we are using schema everywhere.  this is consistent with all implementations 👍